airflow/www/static/js/dag/details/taskInstance/Logs/index.tsx

```diff
@@ -99,12 +98,13 @@ const Logs = ({
   tryNumber,
 }: Props) => {
   const [internalIndexes, externalIndexes] = getLinkIndexes(tryNumber);
-  const [selectedAttempt, setSelectedAttempt] = useState(1);
+  const [manuallySelectedAttempt, setSelectedAttempt] = useState<number | undefined>();
```

Comment:
Not: Should we rename also the setter ? (Not a big fan of having the state and its setter with different names)